arch/risc-v/src/common/riscv_misaligned.c

```diff
@@ -443,9 +518,31 @@ static bool decode_insn(uintptr_t *regs, riscv_insn_ctx_t *ctx)
 
       case INSN_FLW:
       case INSN_FLD:
+
```

Comment:
wrap with `#ifdef CONFIG_ARCH_FPU` is needed.
Also please remove TODO from `/* TODO: Handle float load / store instruction */`

Reply:
Yes, gurad added.